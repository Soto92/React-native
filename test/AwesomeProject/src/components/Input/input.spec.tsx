import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';

import Input from './index';

describe('Input shallow test', () => {
  it('should shallow render a Input component', () => {
    shallow(<Input label="Input" placeholder="placeholder" />);
  });
});

describe('Input Snapshot test', () => {
  it('should render the component correctly', () => {
    const tree = renderer
      .create(<Input label="Input" placeholder="placeholder" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

