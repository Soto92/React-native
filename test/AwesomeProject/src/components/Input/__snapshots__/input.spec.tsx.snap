// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Snapshot test should render the component correctly 1`] = `
<View
  style={
    Object {
      "height": 58,
      "width": 702,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#656975",
        "fontFamily": "Encode Sans",
        "fontSize": 14,
        "fontWeight": "500",
        "marginBottom": 5,
      }
    }
  >
    Input
  </Text>
  <TextInput
    allowFontScaling={true}
    onBlur={[Function]}
    onChangeText={[Function]}
    onFocus={[Function]}
    onSubmitEditing={[Function]}
    placeholder="placeholder"
    rejectResponderTermination={true}
    secureTextEntry={false}
    style={
      Object {
        "borderColor": "#d7d8dc",
        "borderRadius": 4,
        "borderWidth": 1,
        "paddingBottom": 10.5,
        "paddingLeft": 16,
        "paddingTop": 8.5,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
